docs/getting_started/benchmark_owner_demo.md

```diff
@@ -74,124 +74,122 @@ medperf test run \
    --labels_path "medperf_tutorial/mock_chexpert/labels"
 ```
 
-Assuming the test passes, we are ready to submit the MLCubes to the MedPerf server.
+Assuming the test passes successfully, you are ready to submit the MLCubes to the MedPerf server.
 
 ## 4. Submitting the MLCubes
 
 #### How does MedPerf Recognize an MLCube?
 
 {% include "getting_started/shared/mlcube_submission_overview.md" %}
 
-To prepare the files of our three MLCubes, run (make sure you are in MedPerf's root folder):
+To prepare the files of our three MLCubes, run the following command ensuring you are in MedPerf's root folder:
 
 ```bash
 python scripts/package-mlcube.py --mlcube medperf_tutorial/chexpert_prep/mlcube --mlcube-types data-preparator
 python scripts/package-mlcube.py --mlcube medperf_tutorial/xrv_densenet/mlcube --mlcube-types model
 python scripts/package-mlcube.py --mlcube medperf_tutorial/metrics/mlcube --mlcube-types metrics
 ```
 
-For each MLCube, this script will create a new folder in the MLCube directory, named `assets`, containing all the files that should be hosted separately.
+For each MLCube, this script will create a new folder named `deploy` in the MLCube directory. This folder will contain all the files that should be hosted separately.
 
 {% include "getting_started/shared/redirect_to_hosting_files.md" %}
 
 #### Submit the MLCubes
 
-To submit the MLCubes:
+#### Data Preparator MLCube
```

Comment:
The main title "Submit the MLCubes" and the subtitles below "Data preparator MLCube, model mlcube, ..." now have the same headers level. They look weird now, right?